packages/adapter-cloudflare/index.js

```diff
@@ -22,10 +22,6 @@ export default function () {
 
 			const relativePath = posix.relative(tmp, builder.getServerDirectory());
 
-			builder.log.info(
-				`adapter-cloudfare is writing its own _headers file. If you have your own, you should duplicate the headers contained in: ${dest}/_headers`
```

Comment:
If people had done this previously as we advised and now we are appending them, they will end up with duplicates. I don't know how much of an issue that is. Does this need to be considered a breaking change and major release?

Reply:
I think you're right about this being a breaking change. I never thought of that.
Are there any examples of existing configurations and how people get around this now?